metadata-ingestion/docs/sources/powerbi-report-server/powerbi_report_server_recipe.yml

```diff
@@ -0,0 +1,24 @@
+source:
+  type: powerbi-report-server
+  config:
+    # Your Power BI Report Server Windows username
+    username: username
+    # Your Power BI Report Server Windows password
+    password: password
+    # Your Workstation name
+    workstation_name: workstation_name
+    # Your Power BI Report Server host URL, example: localhost:80
+    host_port: host_port
+    # Your alias for Power BI Report Server host URL, example: local_powerbi_report_server
+    server_alias: server_alias
+    # Workspace's dataset environments, example: (PROD, DEV, QA, STAGE)
+    env: DEV
+    # Workspace's dataset environments, example: (PROD, DEV, QA, STAGE)
```

Comment:
It is duplicate of env comments, please update it

Reply:
fix comments